### DISCUSSION TITLE
Fab button in page scroller is overlaid by table sticky column

### DISCUSSION THREAD

**Discussion by 549393092** [quasarframework/quasar#14591] [General - Components / Directives / etc] [answered]
Dear Quasar team,

I did a demo to explain this issue, could yo please have a look?
https://codepen.io/Stanley-549393092/pen/ExLOgZG

As you can see, if you scroll down, the fab button is overlaid by table sticky column.
Could you please guide me what is the proper way to fix it, thanks!

Regards

**Comment by pdanpdan** [comment, marked as answer]
You can add
```sass
.q-page-sticky
  z-index: 1
```

https://codepen.io/pdanpdan/pen/OJZaRvp

There is a small delay until it gets on top - I'll have to check why

**Reply by 549393092** [reply]
@pdanpdan 
I tried to change z-index = 0 for the td and it can fix the issue. Not sure if it's the correct way.
```
  td:first-child
    position: sticky
    left: 0
    z-index: 0
```
https://codepen.io/Stanley-549393092/pen/BaxGGaM

Regards
